src/App/Webhook/PdkWebhookManager.php

```diff
@@ -69,14 +69,7 @@ public function call($input, string $context = self::CONTEXT_WEBHOOK): Response
      */
     public function processWebhook(Request $request): void
     {
-        $requiredPath = parse_url($this->webhooksRepository->getHashedUrl(), PHP_URL_PATH);
-        $logContext   = ['request' => get_object_vars($request)];
-
-        if ($request->getRequestUri() !== $requiredPath) {
-            Logger::error('Webhook received with invalid url', $logContext);
-
-            return;
-        }
+        $logContext = ['request' => get_object_vars($request)];
```

Comment:
Waarom check je de url niet? Nu heeft de hash geen zin meer en mag iedereen webhooks sturen naar andermans website. De hash is voor security.